File: docs/docs/API-Reference/python-examples/workflows-api/example-request-2.py
import requests

url = f"{LANGFLOW_SERVER_URL}/api/v2/workflows/stop"
headers = {"Content-Type": "application/json", "x-api-key": LANGFLOW_API_KEY}
payload = {"job_id": "job_id_1234567890"}

response = requests.post(url, json=payload, headers=headers)
print(response.json())

File: docs/docs/API-Reference/javascript-examples/workflows-api/example-request-2.ts
import axios from 'axios';

const url = `${LANGFLOW_SERVER_URL}/api/v2/workflows/stop`;

const payload = {
  job_id: "job_id_1234567890"
};

const stopWorkflow = async () => {
  try {
    const response = await axios.post(url, payload, {
      headers: {
        'Content-Type': 'application/json',
        'x-api-key': LANGFLOW_API_KEY
      }
    });
    console.log(response.data);
  } catch (error) {
    console.error('Error stopping workflow:', error);
  }
};

stopWorkflow();
